src/transaction_signature_options/sign_with_private_key/mod.rs

```diff
@@ -19,7 +20,10 @@ pub struct SignPrivateKey {
     pub nonce: Option<u64>,
     #[interactive_clap(long)]
     #[interactive_clap(skip_default_input_arg)]
-    pub block_hash: Option<String>,
+    pub block_hash: Option<crate::types::crypto_hash::CryptoHash>,
+    #[interactive_clap(long)]
+    #[interactive_clap(skip_default_input_arg)]
+    pub block_height: Option<u64>,
```

Comment:
Use `near_primitives::types::BlockHeight` instead of primitive `u64` type for `block_height`.

Also, update it below in `input_block_height`

Reply:
Done